custom/ewsghana/urls.py

```diff
@@ -22,8 +22,8 @@
     url(r'^ews_fix_locations/$', 'ews_fix_locations', name='ews_fix_locations'),
     url(r'^ews_add_products_to_locs/$', 'ews_add_products_to_locs', name='ews_add_products_to_locs'),
     url(r'^migrate_email_settings/$', 'migrate_email_settings_view', name='migrate_email_settings'),
-    url(r'^migrate_needs_reminders_field/$', 'migrate_needs_reminders_field',
-        name='migrate_needs_reminders_field'),
+    url(r'^migrate_needs_reminders_field/$', 'delete_connections_field',
```

Comment:
we should probably also update the url regex to avoid confusion, but i won't block on this so we can get this into prod today